### ISSUE TITLE
RPC front page

### ISSUE THREAD

**Issue by petecooper** [textpattern/textpattern-branding#5]
Not sure where best to leave this, but the RPC server front page looks a bit janky.

![Screenshot 2021-03-23 at 12 45 23](https://user-images.githubusercontent.com/226578/112148431-cab3ad80-8bd5-11eb-8b23-2ea522ab51c0.png)

We should consider something a bit more appropriate, even though it's a legacy server and will eventually be repurposed.

**Comment by Bloke** [comment]
Definitely. That odd output's what I mentioned the other day when I was trying to install an old version to run upgrade tests. As far as I recall, the root rpc.textpattern.com never had a front page, just an Apache heartbeat or a link to the /lang directory or something.
